winbuild/build_prepare.py

```diff
@@ -160,18 +159,22 @@ def cmd_msbuild(
         "bins": ["cjpeg.exe", "djpeg.exe"],
     },
     "zlib": {
-        "url": "https://zlib.net/FILENAME",
-        "filename": f"zlib{V['ZLIB_DOTLESS']}.zip",
-        "dir": f"zlib-{V['ZLIB']}",
-        "license": "README",
-        "license_pattern": "Copyright notice:\n\n(.+)$",
+        "url": f"https://github.com/zlib-ng/zlib-ng/archive/refs/tags/{V['ZLIBNG']}.zip",
+        "filename": f"zlib-ng-{V['ZLIBNG']}.zip",
+        "dir": f"zlib-ng-{V['ZLIBNG']}",
+        "license": "LICENSE.md",
+        "patch": {
+            r"CMakeLists.txt": {
+                "set_target_properties(zlib PROPERTIES OUTPUT_NAME zlibstatic${{SUFFIX}})": "set_target_properties(zlib PROPERTIES OUTPUT_NAME zlib)",  # noqa: E501
```

Comment:
If zlib-ng creates zlibstatic.lib by default, then wouldn't it seem helpful to detect that in setup.py?  https://github.com/nulano/Pillow/pull/44

Reply:
Oh, is the concern that zlib-ng might have been built without --zlib-compat?